src/utils/search/tests/project-search.spec.js

```diff
@@ -19,3 +20,8 @@ describe("project search", () => {
     expect(matches).toMatchSnapshot();
   });
 });
+describe("project search - highlightMatches", () => {
+  it("simple", () => {
+    expect(highlightMatches("foo yo", "yo")).toMatchSnapshot();
```

Comment:
The test is failing because 
```js
expect(highlightMatches("foo yo", "yo")).toMatchSnapshot();
```

 should be more like

```js
const lineMatch = { value: 'lets foo and then baa', column: 5, match: 'foo' }
expect(highlightMatches(lineMatch)).toMatchSnapshot();
```

Reply:
ah cool now i understand your function a little better.
not that familiar with writing tests yet. TDD the dream :)

Reply:
will update pr as soon as i can thanks for the feedback.